state/store/initialState.js

```diff
@@ -1,7 +1,7 @@
 const initialState = {
-  appHeader: "Kitty News",
+  appHeader: 'Kitty News',
   articles: [],
   currentArticle: {},
-};
+}
```

Comment:
maybe add `credentials: {}`

Reply:
It works without it, so we decided not to add this one